### PR TITLE
Add default values for more openSUSE and SLES distro versions

### PR DESCRIPTION
**Pull request by tampakrap**

### CODE REVIEW

**Comment by tampakrap** [comment]
I don't think the test failures are related to my changes

File: manifests/globals.pp [hunk 1]
```diff
@@ -94,7 +94,13 @@
     'FreeBSD' => '93',
     'OpenBSD' => '9.3',
     'Suse' => $::operatingsystem ? {
-      'SLES' => '91',
+      'SLES' => $::operatingsystemrelease ? {
+        '11.3' => '91',
```
Comment (hunner) [inline manifests/globals.pp]: This should be the `default` case, for backwards compatibility

Comment (cyberious) [inline manifests/globals.pp]: :+1: after default added

Comment (tampakrap) [inline manifests/globals.pp]: the `default => '91'` would be wrong though eg on SLES11SP2 (and maybe on the upcoming SLES12SP1, I need to check it). How about `default => 'undef'`?

Comment (tampakrap) [inline manifests/globals.pp]: I was wrong, SLES11SP2 also has '91', so I changed the default

Comment (DavidS) [inline manifests/globals.pp]: This line can be removed now? Also, how is this working in future versions of SLES? Will 12.1 have 91 again, or does that need another patch?

Using [versioncmp](https://docs.puppetlabs.com/references/latest/function.html#versioncmp) would make that much nicer.

Comment (tampakrap) [inline manifests/globals.pp]: I took the liberty to use 93 as default, and assign the 11.x distros to 91 through a regexp. Now it covers all the 11.x series (along with the upcoming sle11sp4) and also sle12 and the upcoming sle12sp1

**Pull request merged**